awscli/examples/rum/delete-app-monitor.rst

```diff
@@ -0,0 +1,7 @@
+**To deletes an existing app monitor**
+
+The following ``delete-app-monitor`` deletes an existing app monitor. If the command succeeds, no output is returned. ::
+
+	aws rum delete-app-monitor --name AWSApp
```

Comment:
Same here. All of these indents need to be 4 spaces each and not tab indenting.
For more information: https://aws.github.io/aws-cli/docs_styleguide.html#general

The `--name` option also needs to be on it's own line.